altos-rust/altos-core/src/lib.rs

```diff
@@ -41,8 +41,8 @@ extern crate std;
 #[macro_use]
 extern crate altos_macros;
 
-#[cfg(all(not(test), not(feature="test"), feature="bump_allocator"))]
-extern crate bump_allocator as allocator;
```

Comment:
The `extern crate bump_allocator as allocator;` should be kept in. We choose which allocator to use through the feature flag.